Makefile

```diff
@@ -38,3 +38,17 @@ test-ci: go/lint
 		-- \
 		$(GO_TEST_ARGS) \
 		$(packages)
+
+.PHONY: sdk/update # service=cloud-foo-service commit=true/false
+sdk/update:
+	@if [ $(service) = "cloud-shared" ]; then \
```

Comment:
So when I was looking through some of this with Megan last week, it was very easy to forget to include the `service` and `commit`, and I don't remember the errors being super clear in those cases. Would it be reasonable to check if those parameters are provided with `-z`?

Reply:
Good call!